docs/api/specs/webConnectionProtocol.md

```diff
@@ -0,0 +1,382 @@
+---
+id: webConnectionProtocol
+sidebar_label: Web Connection Protocol
+title: FDC3 Web Connection Protocol (next)
+---
+
+:::info _[@experimental](../fdc3-compliance#experimental-features)_
+
+The FDC3 Web Connection Protocol (WCP) is an experimental feature added to FDC3 in 2.2. Limited aspects of its  design may change in future versions and it is exempted from the FDC3 Standard's normal versioning and deprecation polices in order to facilitate any necessary change.
+
+:::
+
+The FDC3 Web Connection Protocol (WCP) defines the procedure for a web-application to connect to an FDC3 Desktop Agent. The WCP is used to implement a [`getAgent()`](../ref/GetAgent) function in the [`@finos/fdc3` npm module](https://www.npmjs.com/package/@finos/fdc3), which is the recommended way for web applications to connect to a Desktop Agent. This specification details how it retrieves and provides the FDC3 `DesktopAgent` interface object and requirements that Desktop Agents must implement in order to support discovery and connection via `getAgent()`. Please see the [`getAgent` reference document](../ref/GetAgent.md) for its TypeScript definition and related types.
+
+:::tip
+
+The [`@finos/fdc3` npm module](https://www.npmjs.com/package/@finos/fdc3) provides a `getAgent()` implementation which app can use to connect to a Desktop Agent without having to interact with or understand the WCP directly. See [Support Platforms](../supported-platforms) and the [`getAgent()`](../ref/GetAgent) reference page for more details on using `getAgent()` in an application.
+
+:::
+
+The WCP supports both interfaces to web-based Desktop Agents defined in the FDC3 Standard:
+
+- **Desktop Agent Preload**: An 'injected' or 'preloaded' Desktop Agent API implementation, typically provided by an electron (or similar) container or browser extension, which is made available to web applications at `window.fdc3`.
```

Comment:
"electron" should be "Electron"

Reply:
Fixed in be131b655bbc4812c9cfec080fb8ee86ac81cb85